hedera-node/hedera-mono-service/src/test/java/com/hedera/services/store/contracts/precompile/ERC721PrecompilesTest.java

```diff
@@ -1720,6 +1722,11 @@ private Bytes givenMinimalFrameContextWithoutParentUpdater(Bytes nestedPretendAr
                     List.of(new SyntheticTxnFactory.NftExchange(1, token, sender, receiver)),
                     new ArrayList<>() {});
 
+    public static final CryptoTransferWrapper CRYPTO_TRANSFER_TOKEN_WRAPPER =
+            new CryptoTransferWrapper(
+                    new TransferWrapper(Collections.emptyList()),
+                    Collections.singletonList(TOKEN_TRANSFER_WRAPPER));
+
```

Comment:
Same question as in `ERC20PrecompilesTest`:

Can't we directly use the new constants defined in `HTSTestsUtil`, removing the need for the new ones here?

Reply:
done